polar-core/src/polar.lalrpop

```diff
@@ -181,14 +181,6 @@ BuiltinOperator: Operator = {
 };
 
 New: Value = {
-    // Keyword argument constructor.
-    "new" <literal:Spanned<InstanceLiteralTerm>> => {
```

Comment:
Can we rm the definition of `InstanceLiteralTerm` on 166?

Reply:
Yes.

Reply:
Wait... doesn't that mean we can completely get rid of the whole `<"Pattern">`? (Not for this PR, to verify later!). That would be nice.